examples/simple_real_numbers.rs

```diff
@@ -0,0 +1,96 @@
+extern crate openfhe;
+use cxx::{CxxVector, SharedPtr};
+use openfhe::ffi as ffi;
+
+fn main(){
```

Comment:
Braces starting from the new line for consistency

Reply:
Fixed